resources/i18n/ko.json

```diff
@@ -382,7 +382,32 @@
     "Running": "실행중",
     "Finished": "종료",
     "Others": "기타",
-    "Services": "모델 서비스"
+    "Services": "모델 서비스",
+    "StartNewServing": "새 서비스 시작",
+    "ServiceName": "서비스 이름",
+    "OpenToPublic": "앱을 외부에 공개",
```

Comment:
```suggestion
    "OpenToPublic": "공개",
```

Reply:
@yomybaby I just followed [this line](https://github.com/lablup/backend.ai-webui/blob/6b0e4513c681781eb8e849fc2825efaa0e381539/resources/i18n/ko.json#L282). Is it better to change?

Reply:
Okay, I'll merge without this suggestion now and make changes later if necessary. 👍